python/hail/expr/expression.py

```diff
@@ -3118,59 +3173,108 @@ def to_boolean(self):
 class CallExpression(Expression):
     """Expression of type :class:`.TCall`.
 
-    >>> call = functions.capture(Call(1))
+    >>> call = functions.capture(Call([0, 1]))
     """
-    @property
-    def gt(self):
-        """Returns the triangle number of :meth:`.Call.gtj` and :meth:`.Call.gtk`.
+
+    def __getitem__(self, item):
+        """Get the i*th* allele.
 
         Examples
         --------
+
+        Index with a single integer:
+
         .. doctest::
 
-            >>> eval_expr(call.gt)
+            >>> eval_expr(call[1])
             1
 
+            >>> eval_expr(call[0])
+            0
+
+        Parameters
+        ----------
+        item : int or :class:`.Int32Expression`
+            Allele index.
+
         Returns
         -------
         :class:`.Int32Expression`
-            Triangle number of the two alleles.
         """
-        return self._field("gt", TInt32())
+        if isinstance(item, slice):
+            raise NotImplementedError("CallExpression does not support indexing into array with a slice.")
```

Comment:
I think "into array" is confusing here because it is a call.  Just say "indexing with a slice"?